test/gtest/shp/containers.cpp

```diff
@@ -31,16 +31,32 @@ TYPED_TEST(DistributedVectorTest, fill_constructor) {
                     typename TestFixture::LocalVec(10, 1)));
 }
 
-// TODO: add more tests: fill_constructor_one_item, fill_constructor_no_items
+TYPED_TEST(DistributedVectorTest, fill_constructor_one_item) {
+  EXPECT_TRUE(equal(typename TestFixture::DistVec(1),
```

Comment:
this ctor initializes with random value, both vectors may be different
use ctor with explicit value, e.g DistVec(1,77), LocalVec(1, 77)